mlir/include/mlir/Dialect/LLVMIR/LLVMIntrinsicOps.td

```diff
@@ -214,14 +214,14 @@ class LLVM_MemcpyIntrOpBase<string name> :
     /*requiresAccessGroup=*/1, /*requiresAliasAnalysis=*/1> {
   dag args = (ins Arg<LLVM_AnyPointer,"",[MemWrite]>:$dst,
                   Arg<LLVM_AnyPointer,"",[MemRead]>:$src,
-                  AnySignlessInteger:$len, I1Attr:$isVolatile);
```

Comment:
Seems to me that should handle this as we do with the FastMathFlags: that is a enum attribute modeling the bitfields.

Reply:
thanks, looking ...